### PR TITLE
[Cherrypick]Prepare for sca web ui 6.15.z

### PR DESCRIPTION
**Pull request by ColeHiggins2**
Cherrypick of https://github.com/SatelliteQE/robottelo/pull/13004
Issue: https://github.com/SatelliteQE/robottelo/issues/13356

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_activationkey.py::test_positive_prepare_for_sca_only_ak  tests/foreman/ui/test_contenthost.py::test_positive_prepare_for_sca_only_content_host tests/foreman/ui/test_organization.py::test_positive_prepare_for_sca_only_organization tests/foreman/ui/test_subscription.py::test_positive_prepare_for_sca_only_subscription

File: tests/foreman/ui/test_subscription.py [hunk 1]
```diff
@@ -551,3 +551,21 @@ def test_positive_candlepin_events_processed_by_STOMP(
...
+        will be required for all organizations in Satellite 6.16
+
+    :id: cb6fdfdd-04ee-4acb-9460-c78556cef11e
+
+    :expectedresults: The Subscription page notifies users that Simple Content Access will
+        be required for all organizations in Satellite 6.16
+    """
+    with target_sat.ui_session() as session:
+        session.organization.select(function_entitlement_manifest_org.name)
+        sca_alert = session.subscription.sca_alert()
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_subscription.py:567]: @ColeHiggins2 PRT failed with `AttributeError: 'SubscriptionEntity' object has no attribute 'sca_alert'`, I think you'll need to decorate sca_alert with `@property` to fix this in airgun. I think it was missed in your original master branch PR as you didn't run the PRT and it was merged w/o PRT

**Review by Gauravtalreja1 — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_subscription.py (hunk 1 above)
Comment (ColeHiggins2) [reply tests/foreman/ui/test_subscription.py:567]: Just got a fix for this in airgun, lets rerun prt now

**Comment by ColeHiggins2** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_activationkey.py::test_positive_prepare_for_sca_only_ak tests/foreman/ui/test_contenthost.py::test_positive_prepare_for_sca_only_content_host tests/foreman/ui/test_organization.py::test_positive_prepare_for_sca_only_organization tests/foreman/ui/test_subscription.py::test_positive_prepare_for_sca_only_subscription

**Review by vsedmik — APPROVED** [review]
Looks good, PRT 5293 passed.

File: tests/foreman/ui/test_subscription.py (hunk 1 above)
Comment (Gauravtalreja1) [reply tests/foreman/ui/test_subscription.py:567]: just for future ref, PR fixing this https://github.com/SatelliteQE/airgun/pull/1123

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**